packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -88,7 +88,10 @@ describeCompat("Named root data stores", "FullCompat", (getTestObjectProvider) =
 	 * Gets an aliased data store with the given id. Throws an error if the data store cannot be retrieved.
 	 */
 	async function getAliasedDataStoreEntryPoint(dataObject: ITestFluidObject, id: string) {
-		const dataStore = await runtimeOf(dataObject).getAliasedDataStoreEntryPoint(id);
+		// Back compat support - older versions of the runtime do not have getAliasedDataStoreEntryPoint
+		// Can be removed once we no longer support ^2.0.0-internal.7.0.0
+		const dataStore = await (runtimeOf(dataObject).getAliasedDataStoreEntryPoint?.(id) ??
+			(runtimeOf(dataObject) as any).getRootDataStore(id, false /* wait */));
```

Comment:
@agarwal-navin, please leave any thoughts on this, since I believe you worked on removing `getDataStore()`. Also, please let me know if you have any concerns about customers facing this issue when upgrading from 1.X to 2.X. Happy to sync up if you need context as well